package.json

```diff
@@ -483,6 +488,7 @@
         "eslint-plugin-jsdoc": "^48.2.2",
         "js-yaml": "^4.1.0",
         "mocha": "^10.4.0",
+        "npm-run-all": "^4.1.5",
         "prettier": "^3.2.5",
```

Comment:
This is referenced from https://github.com/microsoft/vscode-generator-code/blob/4812b7e/generators/app/templates/ext-command-ts/vscode-esbuild/package.json, but it has 1700+ dependent packages...

Reply:
That does seem like a lot of extra dependency weight (with the attendant impact on CI times and security alert noise) for something of marginal utility.  Developers need to take explicit action to reload the extension anyway.